src/shared/components/RadioButton/RadioButton.style.ts

```diff
@@ -69,35 +77,36 @@ export const StyledInput = styled.div<RadioButtonStyleProps>`
   justify-content: center;
   align-items: center;
   background-clip: content-box;
-  padding: 2px;
+  padding: 3px;
   box-sizing: border-box;
   width: 18px;
   height: 18px;
-  &::before {
-    content: '';
-    top: -8px;
-    bottom: -8px;
-    left: -8px;
-    right: -8px;
-    border-radius: 50%;
-    position: absolute;
-    z-index: -1;
-  }
   ${colorFromProps};
   & + span {
     color: ${(props) => (props.checked ? colors.white : '')};
   }
+  transition: background-color 0.25s ease, border-color 0.25s ease;
+`
+
+export const RadioButtonWrapper = styled.div`
```

Comment:
Do we need that wrapper? I think for other places we just include the helper as part of the label. Not sure if that's the best approach but let's be consistent